node_modules/titanium-sdk/lib/emulators/genymotion.js

```diff
@@ -286,7 +286,10 @@ function findGenymotion(dir, config, callback) {
 	if (!player || !fs.existsSync(player)) {
 		player = path.join(dir, 'player' + exe);
 	}
-	if (player && !fs.existsSync(player)) {
+	if (!fs.existsSync(player) && process.platform != 'win32') {
```

Comment:
You still need to check if `player` is truthy in every `if` statement. The path you are setting is OS X specific, so just compare to `darwin`. Use `!==` instead of `!=`.

```javascript
if (process.platform === 'darwin' && (!player || !fs.existsSync(player))) {
    player = path.join(dir, 'player.app', 'Contents', 'MacOS', 'player');
}
if (player && !fs.existsSync(player)) {
    player = null;
}
```